modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/Slot.java

```diff
@@ -18,19 +18,28 @@
 
 package org.wso2.andes.kernel.slot;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
 import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * This class stores all the data related to a slot
```

Comment:
It would be nice if we explain what is a slot? A brief explanation of how we use it. (BTW we have Slots in Disruptor too.)